lib/components/base/progress/MProgress.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * @description 进度条组件
+ * @author Muzych
+ * @date 2024/03/15 12:04
+ * @version v1.0.0
+ */
+import './progress.css';
+import { ProgressProps } from './index';
+import { MC, Slot } from '../../../types';
+import { leaf, useProgress } from './useProgress.ts';
+import { CSSProperties, useEffect, useMemo, useState } from 'react';
+
+export default function MProgress(_props: ProgressProps & MC & Slot) {
+
+  const props = _props as Required<ProgressProps>;
+  const [value, setValue] = useState(props.value ?? 0);
+  const [max, setMax] = useState(props.max ?? 100);
+
+  useEffect(() => {
+    setValue(props.value ?? 0);
+    setMax(props.max ?? 100);
+  }, [props.value, props.max]);
```

Comment:
should `value` and `max` use two `useEffect`?
Their changes don't seem to affect each other.